content/local-setup.md

```diff
@@ -0,0 +1,283 @@
+# Local Setup
+
+# Lesson Objectives
+
+- Use basic Solana CLI commands
+- Set up a local environment for Solana program development
+- Use Rust and Solana CLI to deploy a Solana program from your local environment
+
+# TL;DR
+
+- To get started with Solana locally, you’ll first need to install Rust and the Solana CLI
+- Once you have Rust and Solana CLI installed, you’ll be able to build and deploy your programs locally using the `cargo build-bpf` and `solana program deploy` commands.
+
+# Overview
+
+To get started with Solana program development, you’ll need to first install Rust and the Solana CLI. Additionally, if you are on a windows computer, it is recommended to use Windows Subsystem for Linux (WSL) to build your Solana Programs.
```

Comment:
We may want to start with something like this:

"This lesson will be slightly different from the others. Instead of covering a lot of ground on how to write a program or interact with the network, this lesson will primarily focus on the admittedly tedious task of setting up your local development environment.

To get started with Solana program development ..."

Reply:
Then something like "We'll start by going over the steps for installing the Rust compile and the Solana CLI in both Windows and macOS. If you have a Windows machine, by all means skip the macOS section and vice versa. Once you've installed Rust and the Solana CLI, we'll walk through some of the most important Solana CLI commands."

Reply:
In the existing paragraph, I would strike the "Additionally, if you are on a windows computer..." and add that to the Setup on Windows section. I'd replace that sentence with something about what the Solana CLI is. They haven't used it before, so a brief sentence explaining what it is would be good.